src/app/shared/modules/selectors/single-value-selector/single-value-selector-modal.component.html

```diff
@@ -17,7 +17,11 @@
         (select)='onSelect($event)'
       >
         <ngx-datatable-column name="value" prop="value" [minWidth]="80"></ngx-datatable-column>
-        <ngx-datatable-column name="name" prop="name"></ngx-datatable-column>
+        <ngx-datatable-column [width]="30" [sortable]="false">
+          <ng-template let-row="row" ngx-datatable-cell-template>
+            {{ row.name + (!!row.comment ? ' (' + row.comment + ')' : '') }}
```

Comment:
better to wrap this logic into a getter method of the component, templates should have the least logic possible